### ISSUE TITLE
pyhaystack.connect failed with Niagara 4.4.73.24

### ISSUE THREAD

**Issue by ChristianTremblay** [ChristianTremblay/pyhaystack#61]
Results in Authentication failure

Using : 
```
from pyhaystack.client.niagara import Niagara4HaystackSession
import requests
from requests.packages.urllib3.exceptions import InsecureRequestWarning
requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
session = Niagara4HaystackSession(uri='https://192.168.1.2', 
                                  username='user', 
                                  password='password',
                                  http_args={'debug':True, 'tls_verify':False})
            
session.about().result
```
works

**Comment by sjlongland** [comment]
I'll bite, where did you get Niagara 14.4 from?

**Comment by ChristianTremblay** [comment]
Oups ! JCI Workbench 14.4 which is Niagara 4.4.73.24

**Comment by ChristianTremblay** [comment]
Fixed in 0.95.6 (develop)
